tensorflow/lite/delegates/nnapi/nnapi_delegate.cc

```diff
@@ -1553,7 +1615,11 @@ bool NNAPIDelegateKernel::Validate(
   switch (builtin_code) {
     case kTfLiteBuiltinAdd: {
       ExpectMaxOpVersion(version, 2, &val_ctx);
-      ExpectIsFloatOrQuant8Operator(context, node, &val_ctx);
+      if (android_sdk_version >= kMinSdkVersionForNNAPI13) {
```

Comment:
NNAPI in Android R requires the fused activation function to be NONE:
https://android.googlesource.com/platform/frameworks/ml/+/refs/heads/master/nn/runtime/include/NeuralNetworks.h#263

Similarly for MUL, SUB, DIV.

Reply:
changes added. Thanks.